contracts/contracts/multisig/Multisig.cairo

```diff
@@ -326,52 +308,45 @@ end
 #
 
 @constructor
-func constructor{
-        syscall_ptr : felt*,
-        pedersen_ptr : HashBuiltin*,
-        range_check_ptr,
-    }(
-        owners_len : felt,
-        owners : felt*,
-        confirmations_required : felt,
-    ):
+func constructor{syscall_ptr : felt*, pedersen_ptr : HashBuiltin*, range_check_ptr}(
+    owners_len : felt, owners : felt*, confirmations_required : felt
+):
+    const lower_bound = 1
+
     with_attr error_message("invalid number of required confirmations"):
-        assert_le(confirmations_required, owners_len)
+        assert_in_range(confirmations_required, lower_bound, owners_len + 1)
+    end
+
+    with_attr error_message("owners has to be unique"):
```

Comment:
nit: Consider making this message consistent with other error messages, where we don't point to what needs to be but what is invalid:

```suggestion
    with_attr error_message("owners not unique"):
```